src/docs/output-targets/custom-elements.md

```diff
@@ -51,6 +51,20 @@ This flag defaults to `false` when omitted from a Stencil configuration file.
 
 > Note: At this time, components created not using JSX may not be automatically defined. This is a known limitation of the API and users should be aware of it
 
+### generateTypeDeclarations
+
+By default, type declaration files (`.d.ts` files) are only generated for the `dist-custom-elements` output target when
```

Comment:
Is it worth noting what the type declarations files are used for? Do we explain these somewhere else? If so, we should link here. If not, I'm not sure this is the right place to explain them for the first time, but it may be a worthwhile thing to explain somewhere else down the road.

Reply:
We _sort of_ do in other parts of the documentation, but agree we should add a formal suggestion (or reference to the TS docs) somewhere on the site. Created STENCIL-400 to track

Reply:
👍